news/2013-10-08-storyboard-showcase-kozato-45nen-no-yukizakura.md

```diff
@@ -13,7 +13,7 @@ Without further ado, I present to you the first showcased map:
 **[Kozato - 45nen no Yukizakura](https://osu.ppy.sh/s/50771) by [Rewrite](https://osu.ppy.sh/u/1161361)**  
 **Storyboard by [yukinohara](https://osu.ppy.sh/u/580107)**
 
-This map was lucky enough to have both a custom skin and storyboard created for it by yukinohara, a very skilled graphic designer. The pink, cherry blossom-themed images subtly move and transition to set a wonderful atmosphere for the music, and the very sleek custom skin ensures that this atmosphere is accented by the hitobjects, rather than harmed.
+This map was lucky enough to have both a custom skin and storyboard created for it by yukinohara, a very skilled graphic designer. The pink, cherry blossom-themed images subtly move and transition to set a wonderful atmosphere for the music, and the very sleek custom skin ensures that this atmosphere is accented by the hit objects, rather than harmed.
```

Comment:
Changing it in the news too?

Reply:
yeah, it's not a matter of writing style and it doesn't change the meaning or anything so I went for it

Reply:
alright, sounds good